### PR TITLE
Migrate IexecCategoryManager to Typescript & Hardhat

### PR DESCRIPTION
**Pull request by zguesmi**
Minimal diff: [Migrate file to Hardhat](https://github.com/iExecBlockchainComputing/PoCo/commit/eb00588e20250a43564fa0eeb7a15ab4549493f0)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 ## vNEXT
-- Migrate IexecAccessors tests Typescript & Hardhat. (#96)
+- Migrate unit test files to Typescript & Hardhat:
+    - IexecAccessors (#96)
+    - IexecCategoryManager (#97)
```
Comment (gfournieriExec) [inline CHANGELOG.md:6]: Do we want this or set the latest PR on top of the list ?

**Review by gfournieriExec — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zguesmi) [reply CHANGELOG.md:6]: Both are fine for me

**Review by jeremyjams — APPROVED** [review]
Good thank you :)

File: CHANGELOG.md (hunk 1 above)
Comment (zguesmi) [reply CHANGELOG.md]: Done in the last commit. Thanks!

**Review by jeremyjams — APPROVED** [review]
Thanks for the updates :+1:

**Pull request merged**